python/src/Mesh_doc.i.in

```diff
@@ -205,7 +205,11 @@ Examples
 >>> simplex = [[0, 1, 2]]
 >>> mesh2d = ot.Mesh(vertices, simplex)
 >>> print(mesh2d.computeSimplicesVolume())
-[0.5]"
+[0.5]
+>>> simplices = [[0, 1, 1], [1, 2, 2], [2, 0, 0]]
+>>> mesh1d_in_2d = ot.Mesh(vertices, simplices)
+>>> print(mesh1d_in_2d.computeSimplicesVolume())
+[1,1,1.41421]"
```

Comment:
the value testing should be moved to the Mesh test file (doctests should not be used as non regression tests)

Reply:
also check getIntrinsicDimension

Reply:
fixed